### ISSUE TITLE
1.7.0: empty files are published

### ISSUE THREAD

**Issue by cmckni3** [yarnpkg/yarn#5929]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

My built TypeScript files are published as empty files to npmjs.org

**If the current behavior is a bug, please provide the steps to reproduce.**

https://github.com/webcat12345/ngx-ui-switch

**What is the expected behavior?**

I expect all of the files uploaded to have the compiled TypeScript code.

**Please mention your node.js, yarn and operating system version.**

node.js: 8.11.2
yarn: 1.7.0
os: Mac OS 10.13.4 High Sierra

**Comment by rally25rs** [comment]
If you run a `yarn pack` does the output .tgz have correct files? In my output, there are no 0 length files:

```
~/Projects/ngx-ui-switch (master) 🐒   yarn pack
yarn pack v1.7.0
success Wrote tarball to "/Users/jvalore/Projects/ngx-ui-switch/ngx-ui-switch-v6.0.3.tgz".
✨  Done in 0.13s.

~/Projects/ngx-ui-switch (master) 🐒   tar -tvf ngx-ui-switch-v6.0.3.tgz | grep -v -G ^d
-rw-r--r--  0 0      0        1093 Jun  5 13:50 package/.all-contributorsrc
-rw-r--r--  0 0      0         110 Jun  5 13:50 package/.codecov.yml
-rw-r--r--  0 0      0         245 Jun  5 13:50 package/.editorconfig
-rw-r--r--  0 0      0          10 Jun  5 13:50 package/.nojekyll
-rw-r--r--  0 0      0         962 Jun  5 13:50 package/.travis.yml
-rw-r--r--  0 0      0        1078 Jun  5 13:50 package/LICENSE
-rw-r--r--  0 0      0        6860 Jun  5 13:50 package/README.md
-rw-r--r--  0 0      0        3631 Jun  5 13:50 package/angular.json
-rw-r--r--  0 0      0        4330 Jun  5 13:50 package/build.ts
-rw-r--r--  0 0      0         968 Jun  5 13:50 package/karma.conf.js
-rw-r--r--  0 0      0          10 Jun  5 13:56 package/ngx-ui-switch-v6.0.3.tgz
-rw-r--r--  0 0      0      497115 Jun  5 13:50 package/package-lock.json
-rw-r--r--  0 0      0        3191 Jun  5 13:50 package/package.json
-rw-r--r--  0 0      0         837 Jun  5 13:50 package/protractor-ci.conf.js
-rw-r--r--  0 0      0         722 Jun  5 13:50 package/protractor.conf.js
-rw-r--r--  0 0      0        3034 Jun  5 13:50 package/tslint.json
-rw-r--r--  0 0      0         386 Jun  5 13:50 package/typedoc.json
-rw-r--r--  0 0      0         302 Jun  5 13:50 package/e2e/app.e2e-spec.ts
-rw-r--r--  0 0      0         212 Jun  5 13:50 package/e2e/app.po.ts
-rw-r--r--  0 0      0         235 Jun  5 13:50 package/e2e/tsconfig.e2e.json
-rw-r--r--  0 0      0        5430 Jun  5 13:50 package/src/favicon.ico
-rw-r--r--  0 0      0         336 Jun  5 13:50 package/src/main.ts
-rw-r--r--  0 0      0        2477 Jun  5 13:50 package/src/polyfills.ts
-rw-r--r--  0 0      0        1344 Jun  5 13:50 package/src/styles.scss
-rw-r--r--  0 0      0         637 Jun  5 13:50 package/src/test.ts
-rw-r--r--  0 0      0         279 Jun  5 13:50 package/src/tsconfig.app.json
-rw-r--r--  0 0      0         324 Jun  5 13:50 package/src/tsconfig.spec.json
-rw-r--r--  0 0      0         176 Jun  5 13:50 package/src/typings.d.ts
-rw-r--r--  0 0      0         172 Jun  5 13:50 package/src/app/app.component.html
-rw-r--r--  0 0      0        1313 Jun  5 13:50 package/src/app/app.component.spec.ts
-rw-r--r--  0 0      0         586 Jun  5 13:50 package/src/app/app.component.ts
-rw-r--r--  0 0      0         976 Jun  5 13:50 package/src/app/app.module.ts
-rw-r--r--  0 0      0           0 Jun  5 13:50 package/src/assets/.gitkeep
-rw-r--r--  0 0      0          51 Jun  5 13:50 package/src/environments/environment.prod.ts
-rw-r--r--  0 0      0         387 Jun  5 13:50 package/src/environments/environment.ts
-rw-r--r--  0 0      0         702 Jun  5 13:50 package/src/lib/package.json
-rw-r--r--  0 0      0          95 Jun  5 13:50 package/src/lib/public_api.ts
-rw-r--r--  0 0      0         741 Jun  5 13:50 package/src/lib/tsconfig-build.json
-rw-r--r--  0 0      0         461 Jun  5 13:50 package/src/lib/tsconfig-esm.json
-rw-r--r--  0 0      0        1682 Jun  5 13:50 package/src/app/demo/demo.component.html
-rw-r--r--  0 0      0         904 Jun  5 13:50 package/src/app/demo/demo.component.spec.ts
-rw-r--r--  0 0      0         551 Jun  5 13:50 package/src/app/demo/demo.component.ts
-rw-r--r--  0 0      0         480 Jun  5 13:50 package/src/app/demo-cdr/demo-cdr.component.html
-rw-r--r--  0 0      0         857 Jun  5 13:50 package/src/app/demo-cdr/demo-cdr.component.spec.ts
-rw-r--r--  0 0      0         420 Jun  5 13:50 package/src/app/demo-cdr/demo-cdr.component.ts
-rw-r--r--  0 0      0         627 Jun  5 13:50 package/src/app/footer/footer.component.spec.ts
-rw-r--r--  0 0      0         643 Jun  5 13:50 package/src/app/footer/footer.component.ts
-rw-r--r--  0 0      0         656 Jun  5 13:50 package/src/app/github-link/github-link.component.spec.ts
-rw-r--r--  0 0      0        1548 Jun  5 13:50 package/src/app/github-link/github-link.component.ts
-rw-r--r--  0 0      0        1388 Jun  5 13:50 package/src/app/header/header.component.html
-rw-r--r--  0 0      0         627 Jun  5 13:50 package/src/app/header/header.component.spec.ts
-rw-r--r--  0 0      0         226 Jun  5 13:50 package/src/app/header/header.component.ts
-rw-r--r--  0 0      0        5340 Jun  5 13:50 package/src/lib/ui-switch/ui-switch.component.ts
-rw-r--r--  0 0      0         183 Jun  5 13:50 package/src/lib/ui-switch/ui-switch.config.ts
-rw-r--r--  0 0      0         767 Jun  5 13:50 package/src/lib/ui-switch/ui-switch.module.ts
-rw-r--r--  0 0      0         124 Jun  5 13:50 package/src/lib/ui-switch/ui-switch.token.ts
```

**Comment by rally25rs** [comment]
(accidentally clicked close and comment. oops.)

**Comment by cmckni3** [comment]
I released `6.0.3` using `npm publish`. Let me check the now "deprecated" versions 6.0.0-6.0.2.

**Comment by cmckni3** [comment]
Hmm, seems like it might actually be an issue with the `package.json` "release" script.

I'm digging in some more.

**Comment by cmckni3** [comment]
Seems to be working now but those packages contain partial uploads for some reason.

**Comment by cmckni3** [comment]
I apologize for giving you the run around. I published on Friday when npmjs.org was having issues.

https://status.npmjs.org/

>>
Missing package versions after apparently successful publishes
Resolved - This incident has been resolved.
Jun 1, 17:06 UTC
Monitoring - We have identified the issue and submitted a fix. We will continue to monitor the situation.
May 31, 17:19 UTC
Investigating - We are investigating reports of missing package versions after what appears like a successful publish from the CLI.
May 31, 16:26 UTC
